libs/designsystem/src/lib/components/list/list-virtual-scroll-settings.type.ts

```diff
@@ -1,25 +1,3 @@
-// These types were copied from the vscroll package internals (vscroll/dist/typings/interfaces/settings.d.ts)
+import { Settings as VirtualScrollerSettings } from 'vscroll/dist/typings/interfaces/settings';
```

Comment:
Can be done as a one-liner, but I am not going to block because of that. 

```javascript
export { Settings as VirtualScrollerSettings } from 'vscroll/dist/typings/interfaces/settings';
```

Therefore! Approved 🚀!!!!!!!